src/main/antlr4/GobraLexer.g4

```diff
@@ -78,4 +78,6 @@ PREDICATE   : 'predicate';
 WRITEPERM   : 'writePerm' -> mode(NLSEMI);
 NOPERM      : 'noPerm' -> mode(NLSEMI);
 TRUSTED     : 'trusted' -> mode(NLSEMI);
-OUTLINE     : 'outline';
\ No newline at end of file
+OUTLINE     : 'outline';
+INIT_POST   : 'initEnsures';
+IMPORT_PRE  : 'importRequires';
```

Comment:
I also find the keywards confusing. To my knowledge, Peter agreed with Joao's terminology, so I would accept it for now and maybe bring up the discussion again at some future meeting.

Reply:
I don't like the idea of `provides` for both because we are overloading that keyword for two very distinct usages, and that may make it confusing to users. Regarding the choice between `provides`, `ensures`, and `initEnsures`, my opinion is that `ensures` should not be used for the same reason as above (to avoid overloading keywords with multiple usages). Additionally, I don't like `provides` because users might perceive it as synonymous of `ensures` and start mixing their usages - again, may be confusing. `initEnsures`, while arguably ugly, is precise. 

Regarding the choice of `importRequires`, when I am verifying a package, I consider an import precondition as a precondition for the initialization of a package - you assume that those properties hold in order to prove that the init postcondition is satisfied.

I'd be happy to discuss other options though.